app/src/main/java/org/wikipedia/captcha/Captcha.kt

```diff
@@ -1,12 +1,13 @@
 package org.wikipedia.captcha
 
+import com.google.gson.annotations.SerializedName
 import kotlinx.serialization.Serializable
 import org.wikipedia.dataclient.mwapi.MwResponse
 
 @Serializable
-data class Captcha(private val fancycaptchareload: FancyCaptchaReload) : MwResponse() {
+data class Captcha(@SerializedName("fancycaptchareload") private val fancyCaptchaReload: FancyCaptchaReload) : MwResponse() {
```

Comment:
We do not use Gson for serialization, therefore the `SerializedName` annotation will have no effect. Please use the correct annotation for kotlinx.serialization.